fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -117,18 +117,20 @@ private void importFile(final File f) throws Exception {
                 response = importContainer(uri, sanitize(model));
             }
 
-            if (response.getStatusCode() > 204) {
+            if (response.getStatusCode() == 401) {
+                throw new AuthenticationRequiredRuntimeException();
+            } else if (response.getStatusCode() > 204 || response.getStatusCode() < 200) {
                 logger.warn("Error while importing {} ({}): {}",
                    f.getAbsolutePath(), response.getStatusCode(), IOUtils.toString(response.getBody()));
             } else {
                 logger.info("Imported {}: {}", f.getAbsolutePath(), uri);
             }
         } catch (FcrepoOperationFailedException ex) {
-            throw new Exception("Error importing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
+            throw new RuntimeException("Error importing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
         } catch (IOException ex) {
-            throw new Exception("Error reading or parsing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
+            throw new RuntimeException("Error reading or parsing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
         } catch (URISyntaxException ex) {
```

Comment:
I disagree. It is useful to catch this (no one is throwing it) because it enables better diagnostic info than what you would get by just allowing 111 to go wrong. And given that arbitrary RDF could be going into this, 111 _could_ fail.